po/git-novice.ja.po

```diff
@@ -462,46 +462,46 @@ msgid ""
 "(and not a substitute for) the [full legal text of the CC BY 4.0\n"
 "license][cc-by-legal]."
 msgstr ""
-"All Software Carpentry and Data Carpentry instructional material is\n"
-"made available under the [Creative Commons Attribution\n"
-"license][cc-by-human]. The following is a human-readable summary of\n"
-"(and not a substitute for) the [full legal text of the CC BY 4.0\n"
-"license][cc-by-legal]."
+"全てのSoftware Carpentry と Data Carpentry の教材は\n"
+"[クリエイティブ・コモンズ・ライセンス（CC BY ライセンス）][cc-by-human]に基づいて\n"
+"提供されています。以下文は、省略された[CC ライセンス][cc-by-legal]の内容です。\n"
+"（省略された内容はオリジナルの[CC BY ライセンス 4.0][cc-by-legal]の内容\n"
+"の代わりではないので、ご注意下さい。）"
 
 #: git-novice/LICENSE.md:14
 msgid "You are free:"
-msgstr "You are free:"
+msgstr "あなたは以下の行動が許されます："
 
 # unordered list
 #: git-novice/LICENSE.md:16
 msgid "* to **Share**---copy and redistribute the material in any medium or format"
-msgstr "* to **Share**---copy and redistribute the material in any medium or format"
+msgstr "* 教材内容を複製、そして配布・共有する事"
 
 # unordered list
 #: git-novice/LICENSE.md:17
 msgid "* to **Adapt**---remix, transform, and build upon the material"
-msgstr "* to **Adapt**---remix, transform, and build upon the material"
+msgstr "* 教材内容を変更、もしくは教材を基に新しい教材を作ることができます"
 
 #: git-novice/LICENSE.md:19
 msgid "for any purpose, even commercially."
-msgstr "for any purpose, even commercially."
+msgstr "（それがたとえ営利目的であっても）"
 
 #: git-novice/LICENSE.md:21
 msgid ""
 "The licensor cannot revoke these freedoms as long as you follow the\n"
 "license terms."
 msgstr ""
-"The licensor cannot revoke these freedoms as long as you follow the\n"
-"license terms."
+"以上の行動は、ライセンスの条件を満たしていれば\n"
```

Comment:
I'm certainly no lawyer, but I think license-related content is one of the few areas where we probably want to stick to as literal a translation as possible, even if the Japanese ends up sounding a little funny.

Here, I think the meaning is fine, but we lose "the licensor cannot revoke"